app/controllers/miq_task_controller.rb

```diff
@@ -347,12 +357,6 @@ def tasks_scopes(opts, use_times = true)
       scope << :no_status_selected
     end
 
-    # Add time scope
-    if use_times
-      t = format_timezone(opts[:time_period].to_i != 0 ? opts[:time_period].days.ago : Time.now, Time.zone, "raw")
-      scope << [:with_updated_on_between, t.beginning_of_day, t.end_of_day]
-    end
-
```

Comment:
Why was this moved up? It's a little hard to understand, but I don't see how it would make a difference.